docs/features/writing-different-telemetry-types.md

```diff
@@ -30,10 +30,51 @@ Dependencies allow you to track how your external dependencies are doing to give
 
 We provide support for the following dependencies:
 
+- [Azure Service Bus](#measuring-azure-service-bus-dependencies)
 - [Custom](#measuring-custom-dependencies)
 - [HTTP](#measuring-http-dependencies)
 - [SQL](#measuring-sql-dependencies)
 
+### Measuring Azure Service Bus dependencies
+
+We allow you to measure Azure Service Bus dependencies for both queues & topics or unknown entities.
```

Comment:
I get what you mean but here it doesn't really make sense if you ask me.
```suggestion
We allow you to measure Azure Service Bus dependencies for both queues & topics.
```

You'll always know the entity but maybe you don't know its type; but don't think it's important to emphasize that here. The reason why I wanted to add this line is clearly state that it both supports queues & topics which are part of the same product.